### PR TITLE
review of google drive functions

### PR DESCRIPTION
**Pull request by philganz**
This merge would:  
- Change one line in functions.R to avoid an error in compare_local_and_gdrive.  
- Make some minor typo corrections and a text clarification in GoogleDriveTest.Rmd.

### CODE REVIEW

File: functions.R [hunk 1]
```diff
@@ -312,9 +312,7 @@ compare_local_and_gdrive <- function(l_path, g_path){
   #' It is also useful because this version of modifiedTime does not change if someone were to edit the file in any way, 
   #' including renaming it to something else and back.  
   drive_rev_info <- googledrive::drive_reveal(g_path$files, what = "published")$revision_resource
-  drive_mtime <- as.POSIXct(sapply(
-    sapply(drive_rev_info, "[[", "modifiedTime"),
-    function(x) as.POSIXct(x, format = "%Y-%m-%dT%H:%M:%OSZ", tz = "GMT"), USE.NAMES = F))
```
Comment (GeoffMayhew-NOAA) [inline functions.R:317]: So you didn't need the date format specified? I'll see if this runs for me (as well as on some other objects).

Comment (philganz) [reply functions.R:317]: Ah, I see now that I chopped off the time with this change. I tried "1960-01-01" and "1960-01-02" for the origin on the outer call to as.POSIXct(), but couldn't seem to get the time component right.

Comment (GeoffMayhew-NOAA) [reply functions.R:317]: What error did you get from the original code?

Comment (philganz) [reply functions.R:317]: I got an error telling me that as.POSIXct() needed 'origin.'

Comment (GeoffMayhew-NOAA) [reply functions.R:317]: I can't reproduce this error. I'm curious which instance of as.POSIXct() is throwing the error for you. Typically, the origin is assumed as "1970-01-01" (not 1960), so I wonder if you're getting a different 'modifiedTime' date format than I am.

If you run lines 187+188 of the markdown (to create your_file_path and gdrive_test_dribble), and then the lines below, what do you get?

```
l_path  <- parse_local_path(your_file_path)
g_path <- parse_dribble(gdrive_test_dribble, l_path)
drive_rev_info <- googledrive::drive_reveal(g_path$files, what = "published")$revision_resource

# The character string of modified times from files in Gdrive
sapply(drive_rev_info, "[[", "modifiedTime")

# Datetimes converted to a numerical form
sapply(
    sapply(drive_rev_info, "[[", "modifiedTime"),
    function(x) as.POSIXct(x, format = "%Y-%m-%dT%H:%M:%OSZ", tz = "GMT"), USE.NAMES = F)
```

Comment (philganz) [reply functions.R:317]: Capture

Comment (GeoffMayhew-NOAA) [reply functions.R:317]: Adding the default origin solved the error. For some reason my computer assumed this origin whereas Phil's didn't.

**Pull request merged**